README.md

```diff
@@ -0,0 +1,95 @@
+# UrContacts
+> A contact list application
+
+UrContacts is a React contact list app built with full CRUD capabilities. It uses json-server to locally store your data in a JSON file at the backend. Because your data persists on a locally hosted database, your data is completely private.
+
+## Installation and Build
+
+1. Clone GitHub repo into a desired local root directory: 
+    - HTTPS: https://github.com/dyazdani/contact-list.git
+    - SSH: `git@github.com:dyazdani/contact-list.git`
+    - GitHub CLI: `gh repo clone dyazdani/contact-list` 
+
+2. `cd contact-list`
+
+3. `npm install` 
+
+4. `npm run build`
+
+5. `npm start`
```

Comment:
Recommend also including the URL & port number where the app will run (I forget which port this one uses? 3000?)